x-pack/platform/plugins/shared/alerting/server/routes/get_alert_deletion_settings.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { IRouter } from '@kbn/core/server';
+import type { ILicenseState } from '../lib';
+import type { AlertingRequestHandlerContext } from '../types';
+import { INTERNAL_BASE_ALERTING_API_PATH } from '../types';
+import type { RewriteResponseCase } from './lib';
+import { verifyAccessAndContext } from './lib';
+import type { RulesSettingsAlertDeletion } from '../../common';
+import { API_PRIVILEGES } from '../../common';
+
+const rewriteBodyRes: RewriteResponseCase<RulesSettingsAlertDeletion> = ({
```

Comment:
do we need a versioned schema for these routes?

Reply:
done in https://github.com/elastic/kibana/pull/211488/commits/bdfa46d4b6df1f33f801d0ba65df8c922e1bfd27 and https://github.com/elastic/kibana/pull/211488/commits/0784d8c7fb8d081e4ad4fa7e75766004d9d0f134